eclipse-scout-core/src/util/files.js

```diff
@@ -48,7 +48,23 @@ export function fileListToArray(fileList) {
   return files;
 }
 
+export function getErrorMessageMaximumUploadSizeExceeded(session, maximumUploadSize) {
+  if (maximumUploadSize < 1024) {
+    return session.text('ui.FileSizeLimit', maximumUploadSize, 'B');
+  }
+  let maximumUploadSizeKB = maximumUploadSize / 1024;
+  if (maximumUploadSizeKB < 1024) {
+    return session.text('ui.FileSizeLimit', maximumUploadSizeKB, 'KB');
```

Comment:
Should you use the Math.ceil/floor function to show a less precise, but prettier number string in the UI? (all three occurrences)